src/store/modules/command_params.js

```diff
@@ -119,6 +125,15 @@ const mutations = {
         // finally, update the declination
         state.mount_dec = val; 
     },
+    mount_ha(state, val) { 
+        let mount_ha = 0
+    },
+    mount_az(state, val) { 
+        let mount_az = 0
+    },
+    mount_alt(state, val) { 
+        let mount_alt = 30
+    },
```

Comment:
I'm not sure what you're trying to do here, and I don't think these functions are modifying anything - if you want to modify the value of `mount_ha` or similar, you'd want to use the same syntax as above, `state.mount_ha = {whatever value you wanted}`, which would allow you to call `mount_ha()`when you wanted to update the value. If your intention is to hardcode defaults, this probably isn't the place to do so - up in the mount parameters would be, though!

Reply:
It is very satisying just to delete things! It still works fine without them. Not sure what I was thinking. Too much of a beginner! I've pressed them all and they work properly. There is an issue with the site-code but thats a different kettle of fish, the signal is getting through :)